src/browser/navigation_validation.ts

```diff
@@ -13,20 +13,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-let coreState = require('./core_state.js');
-let electronApp = require('electron').app;
-let SubScriptionManager:any = require('./subscription_manager.js').SubscriptionManager;
-let subScriptionManager:any = new SubScriptionManager();
+const coreState = require('./core_state.js');
```

Comment:
Agree about the minimum part, I was focusing on a "visual" minimum of a diff. Ok